implementation/src/main/java/io/smallrye/mutiny/groups/MultiOnEvent.java

```diff
@@ -104,7 +106,9 @@ public MultiOnEvent(Multi<T> upstream) {
      * @param action the action to execute when the streams completes, fails or the subscription gets cancelled. Must
      *        not be {@code null}.
      * @return the new {@link Multi}
+     * @deprecated use {{@link Multi#onTermination()}}
      */
+    @Deprecated
     public Multi<T> termination(Runnable action) {
         Runnable runnable = nonNull(action, "action");
         return Infrastructure.onMultiCreation(new MultiSignalConsumerOp<>(
```

Comment:
I would rewrite this to use the new approach and remove this possibility from the operator.